blivet/formats/fs.py

```diff
@@ -1103,7 +1103,7 @@ def needsFSCheck(self):
 class Ext4FS(Ext3FS):
     """ ext4 filesystem. """
     _type = "ext4"
-    _defaultFormatOptions = ["-t", "ext4"]
+    _defaultFormatOptions = ["-F", "-t", "ext4"]
```

Comment:
What about ext2 and ext3? Also, I think this option is worth adding a comment to the code.

Reply:
I have reproduced the issue on ext2 and ext3 as well. Added fixes for these too. I have also added comment to explain why force option is needed.